From 928dc4467650fda5607992288ca4b51d4d489581 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Mon, 15 Sep 2014 14:58:14 +0800
Subject: [PATCH 0167/1594] ENGR00331450-7: ARM: dts: imx6-sabresd: move
 pfuze100 device node to board level dts

commit c0f7845d584f1a36c26b2611422840638cdbb280 from
git://git.freescale.com/imx/linux-2.6-imx.git

Move pfuze100 device node from imx6q/dl-sabresd.dts to imx6qdl-sabresd.dts

Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/boot/dts/imx6dl-sabresd.dts   |    5 -----
 arch/arm/boot/dts/imx6q-sabresd.dts    |    5 -----
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi |    5 +++++
 3 files changed, 5 insertions(+), 10 deletions(-)

diff --git a/arch/arm/boot/dts/imx6dl-sabresd.dts b/arch/arm/boot/dts/imx6dl-sabresd.dts
index 3d1ffed..e9da1b1 100644
--- a/arch/arm/boot/dts/imx6dl-sabresd.dts
+++ b/arch/arm/boot/dts/imx6dl-sabresd.dts
@@ -22,11 +22,6 @@
 	offset-usb-charger = <1285>;
 };
 
-&cpu0 {
-	arm-supply = <&sw1a_reg>;
-	soc-supply = <&sw1c_reg>;
-};
-
 &ldb {
 	lvds-channel@0 {
 		crtc = "ipu1-di0";
diff --git a/arch/arm/boot/dts/imx6q-sabresd.dts b/arch/arm/boot/dts/imx6q-sabresd.dts
index bb0e4f9..44ea024 100644
--- a/arch/arm/boot/dts/imx6q-sabresd.dts
+++ b/arch/arm/boot/dts/imx6q-sabresd.dts
@@ -26,11 +26,6 @@
 	offset-usb-charger = <1685>;
 };
 
-&cpu0 {
-	arm-supply = <&sw1a_reg>;
-	soc-supply = <&sw1c_reg>;
-};
-
 &ldb {
 	lvds-channel@0 {
 		crtc = "ipu2-di0";
diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index 43ce256..6702ad92 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -236,6 +236,11 @@
 	status = "okay";
 };
 
+&cpu0 {
+	arm-supply = <&sw1a_reg>;
+	soc-supply = <&sw1c_reg>;
+};
+
 &ecspi1 {
 	fsl,spi-num-chipselects = <1>;
 	cs-gpios = <&gpio4 9 0>;
-- 
1.7.5.4

